apps/shared/interval_parameter_controller.cpp

```diff
@@ -5,6 +5,7 @@ namespace Shared {
 IntervalParameterController::IntervalParameterController(Responder * parentResponder, InputEventHandlerDelegate * inputEventHandlerDelegate) :
   FloatParameterController<double>(parentResponder),
   m_interval(nullptr),
+  m_tempInterval(),
```

Comment:
Switching line 8 and 7 to avoid compilation warnings as you declared them in the other order in the header file.